examples/custom-resources/externalname-services/transport-server/README.md

```diff
@@ -0,0 +1,101 @@
+# Support for Type ExternalName Services
+
+The Ingress Controller supports routing requests to services of the type [ExternalName](https://kubernetes.io/docs/concepts/services-networking/service/#externalname).
+
+An ExternalName service is defined by an external DNS name that is resolved into the IP addresses, typically external to the cluster. This enables to use the Ingress Controller to route requests to the destinations outside of the cluster.
+
+**Note:** This feature is only available in NGINX Plus.
+
+# Prerequisites
+
+
+For the illustration purpose we will run NGINX Ingress Controller (refered as NIC in the examples) with the ```-watch-namespace=nginx-ingress,default``` option. The option enables NIC to watch selected namespaces.
+
+Any application deployed in other namespaces will be treated as an external service.
+
+We will use the [tls-passthrough](../../tls-passthrough/README.md) application example as our backend app that will be responding to requests.
+
+# Example
+
+## 1. Deploy the tls-passthrough application
+
+1. Deploy the backend application as described in the [tls-passthrough example](../../tls-passthrough/README.md), and make sure it is working as described.
+
+## 2. Deploy external service to external namespace
+
+1. Navigate to the [external-name example](../../../custom-resources/externalname-services/README.md)
+
+2. Deploy external namespace (```external-ns```) and the backend application. Note that the namespace is not being watched by ```NIC```
+    ```
+    $ kubectl apply -f transport-server/secure-app-external.yaml
+    ```
+
+## 3. Setup ExternalName service
+
+1. Refer the newly created service in the file [externalname-svc.yaml](../../../custom-resources/externalname-services/externalname-svc.yaml) in the spec section
+    ```yaml
+    kind: Service
+    apiVersion: v1
+    metadata:
+      name: externalname-service
+    spec:
+      type: ExternalName
+      externalName: secure-app-external-backend-svc.external-ns.svc.cluster.local
+    ```
+
+2. Create the service of type ```ExternalName```
+    ```
+    $ kubectl apply -f externalname-svc.yaml
+    ```
+
+3. Update config map [nginx-config.yaml](../../../custom-resources/externalname-services/nginx-config.yaml) with the resolver address
+    ```yaml
+    kind: ConfigMap
+    apiVersion: v1
+    metadata:
+      name: nginx-config
+      namespace: nginx-ingress
+    data:
+      resolver-addresses: "kube-dns.kube-system.svc.cluster.local"
+    ```
+
+4. Apply the change
+    ```bash
+    $ kubectl apply -f nginx-config.yaml
+    ```
+
+## 4. Change the TS to point to the ExternalName and verify if it is working correctly
+
+1. Navigate to the example [tls-passthrough](../../../custom-resources/tls-passthrough/README.md) and open the ```transport-server.yaml``` file.
```

Comment:
Typo:

```suggestion
1. Navigate to the example [tls-passthrough](../../../custom-resources/tls-passthrough/README.md) and open the ```transport-server-passthrough.yaml``` file.
```
Although honestly I think it would be better to not send users to another directory to modify a file from another example. It's a bit hard to follow.

Reply:
Applied